### PR TITLE
export Event from brigadier and worker's brigadier polyfill

### LINKED ISSUE #1317: Brigadier mistakenly does not export the Event type
This is making it a touch challenging to write `brigade.ts` scripts since `any` type has to be used anywhere the `Event` type should have been used.

### PR DESCRIPTION
**Pull request by krancour**
Fixes #1317

While touching these files anyway, I also organized exports alphabetically, which seems cleaner and more consistent.

Note that there are two new TODOs here that I do not intend to resolve at this time. It seems that approach we used for validating something is exported as expected, which works beautifully for constants, variables, and classes, simply does NOT work for interfaces. I'm not clear on why.

I have, however, manually verified that these changes address #1317.

### CODE REVIEW

**Comment by krancour** [comment]
@brigadecore/maintainers peculiar... after #1318, this new PR didn't automatically request reviews from all the maintainers.

So either there's invalid syntax in `CODEOWNERS` now (according to docs tho, it looks good) _or_ GH simply doesn't request reviews from all individual @brigadecore/maintainers members. If the latter is the case, it's fine by me as long as the branch protection rule that requires approval from a code owner still honors the rights of everyone on @brigadecore/maintainers to approve.

So let's just use this PR as a test case for whether the changes in #1318 are acceptable.

**Review by vdice — APPROVED** [review]

**Pull request merged**
